test/helpers/compiler.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable
+  import/order,
+  no-param-reassign,
+  no-nested-ternary,
+*/
```

Comment:
With the new version of eslint-config-webpack & prettier, you actually don't need the ignores here anymore.